rpa-builder/6.0/modules/ROOT/pages/workbench-toolbar-overview.adoc

```diff
@@ -0,0 +1,213 @@
+
+[[debug-workflow]]
+== Debug Workflow
+With the *Debug Workflow* (image:debug-icon.png[The Debug Workflow icon, 2%, 2%]) button, you can start a debug run of the active Workflow project. The Workflow Debugger automatically starts if you have set break points in your Workflow or if any error occurs during the debug run. Here you can make changes to Workflow elements and retry certain Action Steps or complete Transactions until they are working properly. Then, you can continue to execute the Workflow from the current position without having to start the debug run at the beginning.
+
+// For more information about the Workflow Debugger see Using Workflow Debuger(5.3).
+
+[[print-workflow]]
+== Print
+You can document a Workflow project by clicking *Print Workflow* (image:print-icon.png[The Print icon, 2%, 2%]), which opens the printing function and displays a Print Preview:
+
+image:print-preview.png[The print Workflow preview]
+
+The resulting document shows the steps the Workflow executes during a run, together with the most important settings.
+At the beginning of the window Print Preview there is a menu for further processing the Workflow Print.
+
+image:print-menu.png[The Print Workflow menu]
+
+* *Print...*
++
+Opens a window for starting the print of the Workflow.
+* *Page Setup...*
++
+Opens a window for setting the page format and margins for the print or export of the Workflow.
++
+RPA Builder saves the changes in settings of *Page Setup...* according to your configuration for all users in common or for every user separately and uses these settings for the Workflow Run Protocol and for the Print of a Workflow.
+* *First Page*
++
+Shows the first page.
+* *Previous Page*
++
+Shows the previous page.
+* *Next Page*
++
+Shows the next page.
+* *Last Page*
++
+Shows the last page.
+* *Zoom Out*
++
+Reduces the size of the view by 25%, so that more of the page is visible on screen.
+* *Zoom*
++
+Sets the size of the view to a certain percentage of the original size or to the whole width of the screen or so that a whole page fits on the screen.
+* *Zoom In*
++
+Increases the size of the view by 25% at a time.
+* *Export...*
++
+Opens a window for choosing the file format and a folder for saving to as well as further settings for the export of the print. Allowed formats are PDF, HTML and RTF.
+* *Send...*
++
+Opens a window for choosing the settings for the export of the print as PDF or RTF. The file is attached to an Email automatically after saving.
++
+For the option *Send...*, an Outlook Email profile configured on the computer is required.
+//* *Thumbnails*
+//* *Search* opens a search panel at the lower border of the menu, into which a search expression can be entered. The search can be executed case-sensitive or be restricted to whole words. After pressing the key Enter the first found search result is colored blue. With the buttons Previous resp. Next you can jump to the previous resp. next result. Close closes the search panel.
```

Comment:
```suggestion
//* *Search* opens a search panel at the lower border of the menu, into which a search expression can be entered. The search can be case-sensitive or restricted to whole words. After pressing the Enter key, the first found search result is colored blue. With the buttons Previous resp. Next, you can jump to the previous resp. next result. Close closes the search panel.
```

Reply:
Can these key names be in bold or italics? It is hard to tell what are the names of keys vs regular text.